pkg/cmd/roachprod-microbench/google/service.go

```diff
@@ -84,12 +84,13 @@ func (srv *Service) CreateSheet(
 	var s sheets.Spreadsheet
 	s.Properties = &sheets.SpreadsheetProperties{Title: name}
 
-	// Sort sheets by name.
+	// Sort sheets by name in reverse order. This ensures `sec/op` is the first
+	// sheet and metric in the summary.
```

Comment:
I think I see how this works, but if we add a new column (`watts/sec`) this will quietly break. Are we ok with that? My understanding is that we are since we will eventually be looking at Prometheus as source of truth.